frontend/src/routes/SearchPage/searchDefinitions.tsx

```diff
@@ -1027,9 +1027,10 @@ const searchDefinitions: any = {
                 },
             },
             {
-                header: 'Namespace',
-                sort: 'namespace',
-                cell: 'namespace',
+                header: 'Scope',
+                sort: 'scope',
+                cell: 'scope',
+                tooltip: 'Represents the cluster each PolicyReport is scoped to.',
```

Comment:
@zlayne Can you add a screen capture for context and tag Mikela to review?

Reply:
Hey @dockerymick, Could you review the text in this tooltip? 
![Screen Shot 2021-05-19 at 6 09 05 PM](https://user-images.githubusercontent.com/14047925/118890839-63606400-b8cd-11eb-93c8-05ef5e7a0c25.png)

Reply:
I recommend stating:

Represents the level at which the vulnerabilities were collected from to create the report

Quick comment, can the kind of the policy be renamed from `PolicyReport`. It's confusing bc a policy seems like a report from my viewpoint and there are a ton of policies. Can we be more specific and maybe call it `InsightsPolicy`?

Reply:
@dockerymick We are not able to rename the resource. We are using the public resource found here: https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report/api/v1alpha2.

Reply:
Also not sure if this changes your thinking, but all policyreport resources will exist on the local-cluster (which is why were are not able to use the `cluster:clusterName` filter). So the PolicyReport is not created on the managed cluster.

Reply:
Is the PolicyReport applied to the cluster? 

Maybe, 

Represents the level where the PolicyReport is applied.

Reply:
How about this? "Represents the cluster affected by the PolicyReport."

Reply:
The policyreport is not applied to the cluster - it essentially acts as a summary of the clusters health.

Reply:
Thanks for clarifying and thanks for your suggestion Jorge.

I'm mostly working to avoid using scope to explain `Scope`. Whenever I've seen scope mentioned, it's been used to talk about namespace-scope and cluster-scope, for example in the RBAC doc.

Maybe state:

Represents the area of the cluster where the health logs were pulled from.

Reply:
@dockerymick The PolicyReport resource contains a list of violated policies - which arent exactly the same as health logs. I would be a little bit more comfortable with using something like Jorge posted above. Or something along the lines of:
`Scope refers to the cluster affected by the PolicyReports insights`

For reference this is the info used in the PolicyReport definition of the [scope field](https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/api/v1alpha2/policyreport_types.go#L132):
```
// Scope is an optional reference to the report scope (e.g. a Deployment, Namespace, or Node)
```

Reply:
Thanks Zack, I would suggest: 

"Scope refers to the cluster that is affected by the policy violations reported by the _PolicyReports_ insights."

Is it possible for you all to italicize the word PolicyReports or maybe use monospace for it?

Reply:
The tooltip display only takes a string value currently. So we wont be able to add any text styling at the moment.

Reply:
Represents the cluster that each PolicyReport is scoped to.